static/js/browser2/arc_section.tsx

```diff
@@ -24,11 +24,13 @@ import _ from "lodash";
 
 import { InArcSection } from "./in_arc_section";
 import { OutArcSection } from "./out_arc_section";
+import { PageDisplayType } from "./util";
```

Comment:
fyi, these should be in a separate module, types.ts? they're not really utility... i have a pr coming up to do that.

Reply:
Ok, sounds good!